Backend/package.json

```diff
@@ -8,7 +8,9 @@
     "start": "node ./build/server.js",
     "gcp-build": "tsc -p .",
     "dev:start": "npm-run-all gcp-build start",
-    "dev": "nodemon --watch src -e ts,ejs --exec npm run dev:start"
+    "dev": "nodemon --watch src -e ts,ejs --exec npm run dev:start",
+    "full-backend-local:docker": "docker-compose -f dev/docker-compose.yml up -d",
+    "full-backend-local": "npm-run-all full-backend-local:docker dev"
```

Comment:
After the full start of docker compose we still start "dev" right? I was wondering what is the reasoning about that?

Reply:
The new docker-compose file, that I added, only starts the database + s3, but not the backend server. The backend api is started separately. This way, the backend server can watch code file changes and simply connects to the dockerized db+s3.

Reply:
Ah okay!